scipy/stats/_multivariate.py

```diff
@@ -5864,100 +5889,73 @@ class dirichlet_multinomial_gen(multi_rv_generic):
     def __init__(self, seed=None):
         super().__init__(seed)
         self.__doc__ = doccer.docformat(self.__doc__,
-                                        dirichlet_docdict_params)
+                                        dirichlet_mn_docdict_params)
 
-    def __call__(self, alpha, seed=None):
-        return dirichlet_multinomial_frozen(alpha, seed=seed)
+    def __call__(self, alpha, n, seed=None):
+        return dirichlet_multinomial_frozen(alpha, n, seed=seed)
 
-    def logpmf(self, alpha, x):
+    def logpmf(self, x, alpha, n):
```

Comment:
Lots of changes like this. `x` has to be the first argument, and `n` is a shape parameter of the distribution just as it is for `multinomial`.
![image](https://user-images.githubusercontent.com/6570539/214244366-bee896a4-4c4d-4f4b-96a9-f7911985f45c.png)